docs/config.rst

```diff
@@ -214,6 +258,34 @@ The following configuration shows a bunch of mapping examples:
             </doctrine_mongodb:config>
         </container>
 
+    .. code-block:: php
+
+        use Symfony\Config\DoctrineMongodbConfig;
+
+        return static function (DoctrineMongodbConfig $config): void {
+            $config->documentManager('default')
+                ->mapping('MyBundle1')
+                ->mapping('MyBundle2')
+                    ->type('xml')
+                ->mapping('MyBundle3')
+                    ->type('attribute')
+                    ->dir(__DIR__.'/../src/Document')
```

Comment:
I think the directories should be consistent with previous sections, so this should read `Documents/`

Reply:
Using the constants is one of the advantages of using the PHP configuration, I think it's better to show the full feature as it should be used in the document.